### PR TITLE
Also load up to three fallback layers for 'best quality first'

### LINKED ISSUE #4444: "Best quality first" does not respect disabled "render missing data black" option
When disabling "render missing data black", existing fallback magnifications should be used for rendering. However, this does not work when "best quality first" is enabled.

### PR DESCRIPTION
**Pull request by philippotto**
Right now, the rendering strategies define how many fallback magnifications will be loaded so that these can be rendered. For progressive loading, three fallback magnifications are loaded. For best quality, only one fallback mag was loaded until now. The reasoning behind this was that additional fallbacks shouldn't be necessary (since they are loaded with lower priority). However, this had the effect that missing data couldn't be covered by fallback data if that fallback was more than one mag apart.
I simply changed the value of how many fallback mags should be loaded to three. This can produce more traffic, but since the lower mags have lower priority when using "best quality first", this shouldn't affect the perceived loading speed.

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- Open a dataset for which data only exists in mag 4 or mag 8.
- Select "best quality first" and disable "render missing data black"
- Data should be visible when zooming to mag 1

### Issues:
- fixes #4444 

------
- [x] Updated [changelog](../blob/master/CHANGELOG.md#unreleased)
- ~[ ] Updated [migration guide](../blob/master/MIGRATIONS.md#unreleased) if applicable~
- ~[ ] Updated [documentation](../blob/master/docs) if applicable~
- ~[ ] Adapted [wk-connect](https://github.com/scalableminds/webknossos-connect) if datastore API changes~
- ~[ ] Needs datastore update after deployment~
- [X] Ready for review

### CODE REVIEW

**Comment by MichaelBuessemeyer** [comment]
Looks to me, like this PR also resolves #4329 🧐

File: frontend/javascripts/oxalis/view/right-menu/comment_tab/comment_tab_view.js [hunk 1]
```diff
@@ -94,7 +97,33 @@ type CommentTabState = {
...
 };
 
-class CommentTabView extends React.PureComponent<PropsWithSkeleton, CommentTabState> {
+const RELEVANT_ACTIONS_FOR_COMMENTS = [
+  "updateTree",
+  "deleteTree",
+  "mergeTree",
+  "moveTreeComponent",
+  "deleteEdge",
+  "revertToVersion",
```
Comment (MichaelBuessemeyer) [inline frontend/javascripts/oxalis/view/right-menu/comment_tab/comment_tab_view.js]: It looks like the list is not complete, see my review comment

**Comment by MichaelBuessemeyer** [comment]
btw: Awesome idea to also memoize the tree-daata :rocket:

**Comment by philippotto** [comment]
> Looks to me like this PR also resolves #4329 🧐

Sorry, this was not my intention. I removed these changes again and created a new PR for them. Could you please re-review this PR? :)

**Pull request merged**